scripts/presubmit.sh

```
@@ -15,6 +15,14 @@ if [ ! -f test/cpp/mixer_test.cpp ]; then
   echo "test/cpp/mixer_test.cpp missing — failing fast"
   exit 1
 fi
+if [ ! -f android/app/src/main/cpp/audio_mixer.cpp ]; then
+  echo "android/app/src/main/cpp/audio_mixer.cpp missing — failing fast"
+  exit 1
+fi
 mkdir -p build/cpp_test
-${CXX:-g++} -std=c++17 -Wall -Wextra -pthread test/cpp/mixer_test.cpp -o build/cpp_test/mixer_test
+${CXX:-g++} -std=c++17 -Wall -Wextra -pthread \
+  -I android/app/src/main/cpp \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)